examples/marko-cli/src/components/welcome/index.marko

```diff
@@ -0,0 +1,41 @@
niXsiFfWKcr1/9yqnSHYBzoBjh0+7+k9nUlebF0TOgHsN+aB+ggrl8z7/vHG8ynInOSR8DCvCxkyGmgfYIgnfURNzB+ONZi7q4oD1u85NydJudpjZIiR8VBS5YMTfWr/0bz+hfuEMROgRNvUjtDsoNb4ykJcbEt5Ns175rtHUcG8m3ePhOyiUmxIWoQveNjy5F9a8uTq9H/vJ4ashzYdBKWuzmHSV4zbS17RXA77+B4+xMRMS8jO64U2DZ3Xi2YZTM4JT2bJSyEp3mF9zzxf9CwpXYGtiG3hvM7Zv026Tn1EUpRHGRWi+hXEN+bCKRZ8Itr3mo2Mmt6X/SN5o6t25wUDHLIwDUrVvwsvNX6z5D3S0df5GNeLoW0hPSpIKYAFW9434xO0btAupw5GXm6rDXksmJ1TGNTaRN5ppdj68CBsI2oNBKMUuaisy+bI3mpbghBh9Zwi2wdi99rg6pqqNToipbPTFTeCouqTIPyOETf0Dwt5sCJ7NBmljXMonJVvdTNq5nuUzOebL3NcbHZoMUyx/pCrQAmDJhH72Hlc8d7hFhEMzVByjUDMEgOS4WdL2RexLips5jxX5OuZIvOZKR+tBfC3whqsmbO813bjFcewjoXMetElQqeB4YNmkrf9N9rzjsgAU5mBYcQM/+ZiR5HVS27rry7AgUKFTm0cI68BUhz5OEtIR8I9+eMiruaJzm0wnoOANWAj6QpXiiLsPyaoRi8dg+dcCra3DPi0+4yOhZPXB33RumW3AsXScGddoMafRdmFeNKQsNKAgV0vKslOwH9O/+smghI/ENljJ2AQjlbymtIORW5DgFPumR2BE7hJihql6uGHu6pgz6aN1y5DB2T4j8vYResnmhbQBNGFQaunz627Mp9OvKWpM4Xa3SfM0dBECpmyPq8cTB+1z5REX05xobo6EkjWtjbJOdu5581RDdU+wOTkZTj42ZrIPpQjVzuvZoqRbBI+yih8hrUBhbpkCVffaYxYEd5OLOOcAvBdKvAAej4ezRVJBkp5+cuVH95cBERgmgQA4CoVDOTibhFPJLrXCnv3nTtl1b4DxUyhr4OAKPifL8xOPi3GkVESqVeRXXpfyoCA4Sv/QIa8Im8jSRVte+PQurURtR2wSGqABB05lQye75wk2TvGKs2DsjRR4aGBDyRpBo12ZaBFm/iRxcPZ19IXbAR8wQMoFSTiHuvxVY8dO9237jBwBirO3g9Y0+uXRD0/GMfuCzzLdf1l8BdSubNG/0W/d0LZg80PagA+0SLjBpSXXHvLrjo78/ygGOQPGSUlaqdw8SWVK15W/hXf3z+bRKh+diLno2I4WS4xNSho/IMAWlE6CDu1U2IxrKwvnYomtrtRi6u7kjQZUWVjnPgxCyWoxr76dlzrnlUetktoY0EDiARLOub9nzFAm7/jDhKrGwUd0kB9KuJArlJQJw3ES+/GejP2ZnPRS/wYEjJAVCuUADQDGPtwqcSoq3ZOb72UzbZKsqcDXXjiPuMM2crbHb2yIn7b37Be+CWKIvNxEQskBTARultRiXs8i9RDPbf8PN1ukp8IKO0LOCkxCwq5osqXpYNZRK1sEfDa+9vfcaYfvGtnMAWnPJeahOgCQ1mMRT4OuepfrLs8pxq28gbCStbmGzRVpg7ldEkq23XP0UkNyb+dmgCGT5B/FJGYJVcT5Cah30Yot2v7l7JTfN0RebvpfkvS1T97HFy5kquH7czTD1V2PU198Q++/hror4cJJL5T4lV/GZ7Q9TP0DuRyYmiRqWoI3mndMc3tcXcC90YSTU0QDRJ/EkgTho7v649m7vvXW6s3l5GNfs2/zRdroMTZbwnPsz9e03pjrSS9UjDhIFDIxiezwRehu51w1Zu6z7TjzIuqXVYNOLj33hQll4eN0JQmDUOqRdpZ5YIuEk8rm3mjISW+sE1jKKo9hLZsPPcPXRXL3Xa5OJRNTCp5LTnpDigt8L6VpkuO7D1+6rv1/CS6by8lH6ku1ENKBqtaVanmgT7nQ7o6Cly+2ODl8G8uyQvyvEh+bNulQRdnDaZAmXZ2143nP7A+Tx15zJJSsNvCE1rhwrnORuaPk5S5lOPYY6kCCdKiMuEy92Ty48DdMW65zHrRPEM5xg0nLt9/m61I+f2o3jK6AXg/lRobdBzrZkpLx3HXFdNqCvnlovdEEGcfNHGkDFNC2sCCUWHbHUYs8x7kX5lsJPg6YDcRwCFvENIU6jG5qsipdhX7gKzfS3aYB/VyYW6Imoal/mW762wI3kU62oDKJ0gxTdDSTTj4qf9KXd91S7iunlj/X701k4AEufFzCWFIzjCtNlZRHAJdSCBM8eFtBcHkpM8bconvbzis+WhxmbzSbGLbP3dr8kTa6bJW7/e6bn8xjucxqGGPSQF2AvlB4hHfmDSldi0j3K+/1/i3uifxqz4l7Iq1WJkieJMuah+/8prFjv77P5c+eTV2LDEqVB7iclKlQP7NQ+UdwOs908iTAYCpYIxj0cEW0G6exmeraJsQw2lXS6Sxfs/ARn2YEiMRuu5zfmEgcAG800PmTgLox/UMW0VwpoSpqZyb9Ri6d5QahwuyNhsZloIEP4kALh7oc2aUGO2HqRc+cUXC8H0G3yca4E287dDBAg+kEhOm7bPnfM84x7KfHruL9IHHlFzwAYdOw+j2/Z1sZpvJ7TXW/AiqbVEZCyQJAuz1NlzTbS/xIn5E+V/Qv6DATlU2zfQVje8uTJv6Gado2qIPgMuD5L+oY6LtFeUorn6F2iHuV+eK68l7vayrDn8e6lQ3Dc3o6/23n1esf4+wwa/OxLyJgUS39DDCrFdiM8qivmAN3a6x56wOA9uD+VIZSNvxyhDFA61yJaaZr6IvZ/Ud/iK6hHxHSFkjNf5VNcLPsYIy4jaUczjGumJcZmy1qPIa1i/5V3BrQpUB+yxibrCpsF9dlxOQPLVzw7URcgp7JjL2KTgTWoBkQ0Oq00P8HD2rZq0DBdtAAAAAASUVORK5CYII=');
```

Comment:
OMG 😱

Reply:
I know ☺ . i still need to figure out how to load relative images correctly.